<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Exchange</class>
 <widget class="QWidget" name="Exchange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="topLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer0">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lblSellBanner">
           <property name="text">
            <string>Sells</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lblRoomNameRight">
         <property name="text">
          <string>BBP/DOGE Trading Room</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="lblBuyBanner">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Buys</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Place Order:</string>
         </property>
         <widget class="QLineEdit" name="txtQuantity">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>131</width>
            <height>26</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>Qty</string>
          </property>
          <property name="placeholderText">
           <string>Quantity</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnSell">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>150</y>
            <width>93</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text">
           <string>Sell</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_count_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>85</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Quantity:</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnBuy">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>150</y>
            <width>93</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text">
           <string>Buy</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="txtPrice">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>110</y>
            <width>151</width>
            <height>26</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>Price</string>
          </property>
          <property name="placeholderText">
           <string>Price</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_count_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>85</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Price:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="txtID">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>230</y>
            <width>151</width>
            <height>26</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>ID</string>
          </property>
          <property name="placeholderText">
           <string>ID</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_count_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>210</y>
            <width>85</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>ID:</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnCancel">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>228</y>
            <width>93</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTableWidget" name="tableSell">
         <property name="editTriggers">
          <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Price (...)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantity (...)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Total</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Age</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Flags</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QTableWidget" name="tableBuy">
         <property name="editTriggers">
          <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Price (...)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Qty (...)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Total</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Age</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Flags</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>356</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
